scripts/ci/test_plan.py

```diff
@@ -110,11 +111,11 @@ def get_plan(self, options, integration=False):
 
         logging.info(" ".join(cmd))
         _ = subprocess.call(cmd)
-        with open(fname, newline='') as jsonfile:
-            json_data = json.load(jsonfile)
-            suites = json_data.get("testsuites", [])
-            self.all_tests.extend(suites)
         if os.path.exists(fname):
```

Comment:
what are the cases where this file is not available?

Reply:
When we run tests plan for a board thật is not supported

Reply:
ok, but your patch will just silently skip errors and continue, this is not what we want, the failure here is correct, I mean we can make it nicer, check for the file and abort with an error message, but what you are doing above, will jus continue despite the failure AFAIK

Reply:
yeah, I can add an error message

Reply:
There’s actually an error message already saying “unrecognized platform”, before the exception traceback. So perhaps `subprocess.call` could be replaced with `subprocess.check_call` to capture the exit status of invoking twister and abort early if that fails.